### PR TITLE
SRE-649 build: Remove Nexus use in Ubuntu build

### PR DESCRIPTION
**Pull request by brianjmurrell**
Also use origin/master as reference point in githooks as not everyone
keeps their local master tracking origin/master.

* [ ] Two review approvals and any prior change requests have been resolved.
* [x] Testing is complete and all tests passed or there is a reason documented in the PR why it should be force landed and forced-landing tag is set.
* [x] `Features:` (or `Test-tag*`) commit pragma was used or there is a reason documented that there are no appropriate tags for this PR.
* [x] Commit messages follows the guidelines outlined [here](https://daosio.atlassian.net/wiki/spaces/DC/pages/11133911069/Commit+Comments).
* [x] Any tests skipped by the ticket being addressed have been run and passed in the PR.

### Gatekeeper:

* [ ] You are the appropriate gatekeeper to be landing the patch.
* [ ] The PR has 2 reviews by people familiar with the code, including appropriate watchers.
* [ ] Githooks were used. If not, request that user install them and check copyright dates.
* [ ] Checkpatch issues are resolved.  Pay particular attention to ones that will show up on future PRs.
* [ ] All builds have passed.  Check non-required builds for any new compiler warnings.
* [ ] Sufficient testing is done. Check feature pragmas and test tags and that tests skipped for the ticket are run and now pass with the changes.
* [ ] If applicable, the PR has addressed any potential version compatibility issues.
* [ ] Check the target branch.   If it is master branch, should the PR go to a feature branch?  If it is a release branch, does it have merge approval in the JIRA ticket.
* [ ] Extra checks if forced landing is requested
  * [ ] Review comments are sufficiently resolved, particularly by prior reviewers that requested changes.
  * [ ] No new NLT or valgrind warnings.  Check the classic view.
  * [ ] Quick-build or Quick-functional is not used.
* [ ] Fix the commit message upon landing. Check the standard [here](https://daosio.atlassian.net/wiki/spaces/DC/pages/11133911069/Commit+Comments). Edit it to create a single commit. If necessary, ask submitter for a new summary.

### CODE REVIEW

**Review by JohnMalmberg — APPROVED** [review]

**Review by mjmac — APPROVED** [review]

**Comment by brianjmurrell** [comment]
@daos-stack/daos-gatekeeper:
_Testing is complete and all tests passed or there is a reason documented in the PR why it should be force landed and forced-landing tag is set._
- This PR doesn't affect testing so all testing was skipped in an effort to save wasted CI cycles and expedite landing this issue affecting users in CI.

_Features: (or Test-tag*) commit pragma was used or there is a reason documented that there are no appropriate tags for this PR._
- Per the above reasoning, no testing is being done on this change.

File: utils/rpms/packaging/Dockerfile.ubuntu.20.04 [hunk 1]
```diff
@@ -17,10 +17,12 @@ RUN apt-get update && DEBIAN_FRONTEND=noninteractive apt-get install -y \
             autoconf bash ca-certificates curl debhelper dh-make        \
             dpkg-dev dh-python doxygen gcc git git-buildpackage locales \
             make patch pbuilder pkg-config python3-dev python3-distro   \
-            python3-distutils rpm scons wget
+            python3-distutils rpm scons wget cmake valgrind
```
Comment (ashleypittman) [inline utils/rpms/packaging/Dockerfile.ubuntu.20.04:20]: Why does this PR need to change the packaging list?

Comment (brianjmurrell) [reply utils/rpms/packaging/Dockerfile.ubuntu.20.04:20]: Just to be up-to-date with the packaging [pseudo-]submodule: https://github.com/daos-stack/packaging/blob/456c868721e4518215e01c0beeb6d93ef54d0d19/Dockerfile.ubuntu.20.04#L20

**Comment by brianjmurrell** [comment]
@daos-stack/daos-gatekeeper Please DO NOT land this.  The PR that this was extracted from is doing it's for-landing test run in CI now and I don't want to have to rebase it on this PR once it's done.

**Pull request closed without merging**